.github/workflows/dependency-check.yml

```diff
@@ -0,0 +1,32 @@
+name: Suppressions
+
+on:
+  schedule:
+    # At 12 midnight and every 3rd hour from 10am through 11pm
+    - cron: "0 0,10-23/3 * * *"
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: tibdex/github-app-token@v1
+        id: generate-token
+        with:
+          app_id: ${{ secrets.HMCTS_GITHUB_EXUI_APP_ID }}
+          private_key: ${{ secrets.HMCTS_GITHUB_EXUI_PRIVATE_KEY }}
+      - uses: actions/checkout@v3
+        with:
+          token: ${{ steps.generate-token.outputs.token }}
+          ref: master
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 16
+      - name: Run dependencyCheck
```

Comment:
name could be more specific to updating the known audit suppressions. This will be displayed on Github actions view, and it would be better to have a more specific name displayed there that tells what it is specifically doing.